apps/aechannel/src/aesc_fsm.erl

```diff
@@ -2791,7 +2791,7 @@ send_recoverable_error_msg(?WDRAW_ERR, Sn, Msg) ->
     aesc_session_noise:wdraw_error(Sn,Msg);
 send_recoverable_error_msg(?SHUTDOWN_ERR, Sn, Msg) ->
     case was_fork_activated(?LIMA_PROTOCOL_VSN) of
-        true ->
+        false ->
```

Comment:
This makes no sense (and the tests seem to agree?!) The comment in the next clause indicates that this is backwards, or just wrong...